Tryouts/Messaging-JS/src/InvokeOptions.ts

```diff
@@ -0,0 +1,6 @@
+import { MessageScope } from ".";
```

Comment:
Please Don't import modules like` from "."` and ` from ".."` (this applies for every occurrence, but I won't repeat myself everywhere)

Reply:
Yeah I wasn't sure about that. Should I really import everything from its own file? Does it still make sense to have and index.ts which re-exports everything? I did that so that consuming packages can import everything from a few modules.